apps/cricketscoreapi/.github/workflows/ci.yml

```
@@ -0,0 +1,74 @@
+name: 🏗️ Build & Lint
+
+on:
+  push:
+    branches: [ main, develop ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)